hack/examples/patches/nutanix/csi.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 D2iQ, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+- op: "add"
+  path: "/spec/topology/variables/0/value/addons/csi"
+  value:
+    defaultStorage:
+      providerName: nutanix
+      storageClassConfigName: nutanix-volume
+    providers:
+      - name: nutanix
+        storageClassConfig:
+        - name: nutanix-volume
+          parameters:
+            storageContainer: ${NUTANIX_STORAGE_CONTAINER_NAME}
```

Comment:
This is the only required `parameters` that needs to come from the user.

Reply:
`NutanixVolumes` since this is the default in the CSI driver and what is required for DKP applications.